File: src/fixtures.zig
//! Deterministic first-pass data. Canonical records remain pixel-free; week
//! blocks are an explicit presentation projection used by the prototype.

const calendar = @import("domain/calendar.zig");
const ranking = @import("domain/surface_ranking.zig");

pub const EventTone = enum { accent, surface, muted };

pub const WeekDayHeader = struct {
    id: u64,
    name: []const u8,
    day: u8,
    today: bool,
};

pub const HourRow = struct {
    id: u64,
    label: []const u8,
};

pub const WeekEventBlock = struct {
    event_id: u64,
    calendar_id: u64,
    day_index: u8,
    gap_before: f32,
    height: f32,
    title: []const u8,
    time_label: []const u8,
    location: []const u8,
    tone: EventTone,
    pinned: bool,
};

pub const MonthCell = struct {
    id: u64,
    day: u8,
    in_month: bool,
    today: bool,
    has_event1: bool,
    event1: []const u8,
    has_event2: bool,
    event2: []const u8,
};

pub const fixture_now_ms: i64 = 1_784_120_400_000;

pub const calendar_visibility = [_]calendar.CalendarVisibility{
    .{ .id = 1, .name = "Work", .color = .indigo, .visible = true },
    .{ .id = 2, .name = "Personal", .color = .plum, .visible = true },
    .{ .id = 3, .name = "Focus", .color = .sea, .visible = true },
    .{ .id = 4, .name = "Reminders", .color = .sand, .visible = false },
};

pub const calendar_events = [_]calendar.CalendarEvent{
    .{ .id = 101, .calendar_id = 1, .title = "Weekly planning", .location = "Desk", .starts_at_ms = 1_783_947_600_000, .ends_at_ms = 1_783_951_200_000, .timing = .timed, .event_kind = .meeting, .participation = .accepted, .availability = .busy, .surfaces = .{ .menu_bar = .hide, .desktop_widget = .hide, .up_next = .hide } },
    .{ .id = 102, .calendar_id = 1, .title = "Design review", .location = "Studio", .starts_at_ms = 1_784_037_600_000, .ends_at_ms = 1_784_041_200_000, .timing = .timed, .event_kind = .meeting, .participation = .accepted, .availability = .busy, .surfaces = .{ .menu_bar = .inherit, .desktop_widget = .inherit, .up_next = .inherit } },
    .{ .id = 103, .calendar_id = 3, .title = "Deep work — calendar layout", .location = "Focus mode", .starts_at_ms = 1_784_124_000_000, .ends_at_ms = 1_784_131_200_000, .timing = .timed, .event_kind = .focus, .participation = .accepted, .availability = .busy, .surfaces = .{ .menu_bar = .pin, .desktop_widget = .pin, .up_next = .pin } },
    .{ .id = 104, .calendar_id = 2, .title = "Dentist", .location = "Greenpoint Dental", .starts_at_ms = 1_784_138_400_000, .ends_at_ms = 1_784_142_000_000, .timing = .timed, .event_kind = .appointment, .participation = .accepted, .availability = .busy, .surfaces = .{ .menu_bar = .inherit, .desktop_widget = .inherit, .up_next = .inherit } },
    .{ .id = 105, .calendar_id = 1, .title = "Project review", .location = "Meet", .starts_at_ms = 1_784_210_400_000, .ends_at_ms = 1_784_214_000_000, .timing = .timed, .event_kind = .meeting, .participation = .tentative, .availability = .busy, .surfaces = .{ .menu_bar = .inherit, .desktop_widget = .inherit, .up_next = .inherit } },
    .{ .id = 106, .calendar_id = 3, .title = "Ship the first native pass", .location = "Focus mode", .starts_at_ms = 1_784_293_200_000, .ends_at_ms = 1_784_300_400_000, .timing = .timed, .event_kind = .focus, .participation = .accepted, .availability = .busy, .surfaces = .{ .menu_bar = .inherit, .desktop_widget = .pin, .up_next = .inherit } },
    .{ .id = 107, .calendar_id = 2, .title = "Dinner with Maya", .location = "Win Son", .starts_at_ms = 1_784_329_200_000, .ends_at_ms = 1_784_336_400_000, .timing = .timed, .event_kind = .personal, .participation = .accepted, .availability = .free, .surfaces = .{ .menu_bar = .hide, .desktop_widget = .inherit, .up_next = .hide } },
    .{ .id = 108, .calendar_id = 2, .title = "Yoga", .location = "McCarren Park", .starts_at_ms = 1_784_383_200_000, .ends_at_ms = 1_784_386_800_000, .timing = .timed, .event_kind = .personal, .participation = .accepted, .availability = .free, .surfaces = .{ .menu_bar = .inherit, .desktop_widget = .inherit, .up_next = .inherit } },
    .{ .id = 109, .calendar_id = 3, .title = "Weekly reset", .location = "Home", .starts_at_ms = 1_784_494_800_000, .ends_at_ms = 1_784_498_400_000, .timing = .timed, .event_kind = .task, .participation = .accepted, .availability = .busy, .surfaces = .{ .menu_bar = .inherit, .desktop_widget = .inherit, .up_next = .pin } },
    .{ .id = 110, .calendar_id = 4, .title = "Pay card", .location = "", .starts_at_ms = 1_784_073_600_000, .ends_at_ms = 1_784_160_000_000, .timing = .all_day, .event_kind = .task, .participation = .accepted, .availability = .free, .surfaces = .{ .menu_bar = .hide, .desktop_widget = .hide, .up_next = .hide } },
};

pub const widget_rule: ranking.SurfaceRule = .{
    .surface = .desktop_widget,
    .max_events = 3,
    .horizon_minutes = 10_080,
    .include_all_day = false,
    .include_declined = false,
};

pub const week_days_previous = [_]WeekDayHeader{
    dayHeader(1, "MON", 6, false),  dayHeader(2, "TUE", 7, false),  dayHeader(3, "WED", 8, false),
    dayHeader(4, "THU", 9, false),  dayHeader(5, "FRI", 10, false), dayHeader(6, "SAT", 11, false),
    dayHeader(7, "SUN", 12, false),
};

pub const week_days_current = [_]WeekDayHeader{
    dayHeader(1, "MON", 13, false), dayHeader(2, "TUE", 14, false), dayHeader(3, "WED", 15, true),
    dayHeader(4, "THU", 16, false), dayHeader(5, "FRI", 17, false), dayHeader(6, "SAT", 18, false),
    dayHeader(7, "SUN", 19, false),
};

pub const week_days_next = [_]WeekDayHeader{
    dayHeader(1, "MON", 20, false), dayHeader(2, "TUE", 21, false), dayHeader(3, "WED", 22, false),
    dayHeader(4, "THU", 23, false), dayHeader(5, "FRI", 24, false), dayHeader(6, "SAT", 25, false),
    dayHeader(7, "SUN", 26, false),
};

pub const hour_rows = [_]HourRow{
    .{ .id = 6, .label = "6 AM" },   .{ .id = 7, .label = "7 AM" },   .{ .id = 8, .label = "8 AM" },   .{ .id = 9, .label = "9 AM" },
    .{ .id = 10, .label = "10 AM" }, .{ .id = 11, .label = "11 AM" }, .{ .id = 12, .label = "12 PM" }, .{ .id = 13, .label = "1 PM" },
    .{ .id = 14, .label = "2 PM" },  .{ .id = 15, .label = "3 PM" },  .{ .id = 16, .label = "4 PM" },  .{ .id = 17, .label = "5 PM" },
    .{ .id = 18, .label = "6 PM" },  .{ .id = 19, .label = "7 PM" },  .{ .id = 20, .label = "8 PM" },  .{ .id = 21, .label = "9 PM" },
};

pub const week_event_blocks = [_]WeekEventBlock{
    weekBlock(101, 1, 0, 216, 72, "Weekly planning", "9:00–10:00", "Desk", .surface, false),
    weekBlock(102, 1, 1, 288, 72, "Design review", "10:00–11:00", "Studio", .surface, false),
    weekBlock(103, 3, 2, 288, 132, "Deep work — calendar layout", "10:00–12:00", "Focus mode", .accent, true),
    weekBlock(104, 2, 2, 144, 72, "Dentist", "2:00–3:00", "Greenpoint Dental", .muted, false),
    weekBlock(105, 1, 3, 288, 72, "Project review", "10:00–11:00 · tentative", "Meet", .surface, false),
    weekBlock(106, 3, 4, 216, 132, "Ship the first native pass", "9:00–11:00", "Focus mode", .accent, true),
    weekBlock(107, 2, 4, 576, 96, "Dinner with Maya", "7:00–9:00", "Win Son", .muted, false),
    weekBlock(108, 2, 5, 288, 72, "Yoga", "10:00–11:00", "McCarren Park", .muted, false),
    weekBlock(109, 3, 6, 792, 72, "Weekly reset", "5:00–6:00", "Home", .surface, true),
};

pub const month_cells = [_]MonthCell{
    monthCell(1, 29, false, false, "", ""),                                monthCell(2, 30, false, false, "", ""),                        monthCell(3, 1, true, false, "", ""),
    monthCell(4, 2, true, false, "", ""),                                  monthCell(5, 3, true, false, "Independence Day observed", ""), monthCell(6, 4, true, false, "", ""),
    monthCell(7, 5, true, false, "", ""),                                  monthCell(8, 6, true, false, "", ""),                          monthCell(9, 7, true, false, "Call Mom", ""),
    monthCell(10, 8, true, false, "", ""),                                 monthCell(11, 9, true, false, "Writing block", ""),            monthCell(12, 10, true, false, "", ""),
    monthCell(13, 11, true, false, "", ""),                                monthCell(14, 12, true, false, "", ""),                        monthCell(15, 13, true, false, "Weekly planning", ""),
    monthCell(16, 14, true, false, "Design review", ""),                   monthCell(17, 15, true, true, "Deep work", "Dentist"),         monthCell(18, 16, true, false, "Project review", ""),
    monthCell(19, 17, true, false, "Ship first pass", "Dinner with Maya"), monthCell(20, 18, true, false, "Yoga", ""),                    monthCell(21, 19, true, false, "Weekly reset", ""),
    monthCell(22, 20, true, false, "", ""),                                monthCell(23, 21, true, false, "Research", ""),                monthCell(24, 22, true, false, "", ""),
    monthCell(25, 23, true, false, "Therapy", ""),                         monthCell(26, 24, true, false, "", ""),                        monthCell(27, 25, true, false, "Farmers market", ""),
    monthCell(28, 26, true, false, "", ""),                                monthCell(29, 27, true, false, "Plan August", ""),             monthCell(30, 28, true, false, "", ""),
    monthCell(31, 29, true, false, "Deep work", ""),                       monthCell(32, 30, true, false, "", ""),                        monthCell(33, 31, true, false, "Month review", ""),
    monthCell(34, 1, false, false, "", ""),                                monthCell(35, 2, false, false, "", ""),
};

fn dayHeader(id: u64, name: []const u8, day: u8, today: bool) WeekDayHeader {
    return .{ .id = id, .name = name, .day = day, .today = today };
}

fn weekBlock(id: u64, calendar_id: u64, day_index: u8, gap: f32, height: f32, title: []const u8, time_label: []const u8, location: []const u8, tone: EventTone, pinned: bool) WeekEventBlock {
    return .{ .event_id = id, .calendar_id = calendar_id, .day_index = day_index, .gap_before = gap, .height = height, .title = title, .time_label = time_label, .location = location, .tone = tone, .pinned = pinned };
}

fn monthCell(id: u64, day: u8, in_month: bool, today: bool, event1: []const u8, event2: []const u8) MonthCell {
    return .{ .id = id, .day = day, .in_month = in_month, .today = today, .has_event1 = event1.len > 0, .event1 = event1, .has_event2 = event2.len > 0, .event2 = event2 };
}

File: src/fixtures.ts
import { asciiBytes } from "@native-sdk/core";
import type {
  Bytes,
  CalendarEvent,
  CalendarVisibility,
} from "./domain/calendar.ts";
import type { SurfaceRule } from "./domain/surface-ranking.ts";

export type EventTone = "accent" | "surface" | "muted";

export interface WeekDayHeader {
  readonly id: number;
  readonly name: Bytes;
  readonly day: number;
  readonly today: boolean;
}

export interface HourRow {
  readonly id: number;
  readonly label: Bytes;
}

/** A bounded presentation projection; canonical events remain pixel-free. */
export interface WeekEventBlock {
  readonly eventId: number;
  readonly calendarId: number;
  readonly dayIndex: number;
  readonly gapBefore: number;
  readonly height: number;
  readonly title: Bytes;
  readonly timeLabel: Bytes;
  readonly location: Bytes;
  readonly tone: EventTone;
  readonly pinned: boolean;
}

export interface MonthCell {
  readonly id: number;
  readonly day: number;
  readonly inMonth: boolean;
  readonly today: boolean;
  readonly hasEvent1: boolean;
  readonly event1: Bytes;
  readonly hasEvent2: boolean;
  readonly event2: Bytes;
}

export const FIXTURE_NOW_MS = 1784120400000;

export const CALENDAR_VISIBILITY: readonly CalendarVisibility[] = [
  { id: 1, name: asciiBytes("Work"), color: "indigo", visible: true },
  { id: 2, name: asciiBytes("Personal"), color: "plum", visible: true },
  { id: 3, name: asciiBytes("Focus"), color: "sea", visible: true },
  { id: 4, name: asciiBytes("Reminders"), color: "sand", visible: false },
];

export const CALENDAR_EVENTS: readonly CalendarEvent[] = [
  {
    id: 101,
    calendarId: 1,
    title: asciiBytes("Weekly planning"),
    location: asciiBytes("Desk"),
    startsAtMs: 1783947600000,
    endsAtMs: 1783951200000,
    timing: "timed",
    eventKind: "meeting",
    participation: "accepted",
    availability: "busy",
    surfaces: { menuBar: "hide", desktopWidget: "hide", upNext: "hide" },
    source: null,
  },
  {
    id: 102,
    calendarId: 1,
    title: asciiBytes("Design review"),
    location: asciiBytes("Studio"),
    startsAtMs: 1784037600000,
    endsAtMs: 1784041200000,
    timing: "timed",
    eventKind: "meeting",
    participation: "accepted",
    availability: "busy",
    surfaces: { menuBar: "inherit", desktopWidget: "inherit", upNext: "inherit" },
    source: null,
  },
  {
    id: 103,
    calendarId: 3,
    title: asciiBytes("Deep work — calendar layout"),
    location: asciiBytes("Focus mode"),
    startsAtMs: 1784124000000,
    endsAtMs: 1784131200000,
    timing: "timed",
    eventKind: "focus",
    participation: "accepted",
    availability: "busy",
    surfaces: { menuBar: "pin", desktopWidget: "pin", upNext: "pin" },
    source: null,
  },
  {
    id: 104,
    calendarId: 2,
    title: asciiBytes("Dentist"),
    location: asciiBytes("Greenpoint Dental"),
    startsAtMs: 1784138400000,
    endsAtMs: 1784142000000,
    timing: "timed",
    eventKind: "appointment",
    participation: "accepted",
    availability: "busy",
    surfaces: { menuBar: "inherit", desktopWidget: "inherit", upNext: "inherit" },
    source: null,
  },
  {
    id: 105,
    calendarId: 1,
    title: asciiBytes("Project review"),
    location: asciiBytes("Meet"),
    startsAtMs: 1784210400000,
    endsAtMs: 1784214000000,
    timing: "timed",
    eventKind: "meeting",
    participation: "tentative",
    availability: "busy",
    surfaces: { menuBar: "inherit", desktopWidget: "inherit", upNext: "inherit" },
    source: null,
  },
  {
    id: 106,
    calendarId: 3,
    title: asciiBytes("Ship the first native pass"),
    location: asciiBytes("Focus mode"),
    startsAtMs: 1784293200000,
    endsAtMs: 1784300400000,
    timing: "timed",
    eventKind: "focus",
    participation: "accepted",
    availability: "busy",
    surfaces: { menuBar: "inherit", desktopWidget: "pin", upNext: "inherit" },
    source: null,
  },
  {
    id: 107,
    calendarId: 2,
    title: asciiBytes("Dinner with Maya"),
    location: asciiBytes("Win Son"),
    startsAtMs: 1784329200000,
    endsAtMs: 1784336400000,
    timing: "timed",
    eventKind: "personal",
    participation: "accepted",
    availability: "free",
    surfaces: { menuBar: "hide", desktopWidget: "inherit", upNext: "hide" },
    source: null,
  },
  {
    id: 108,
    calendarId: 2,
    title: asciiBytes("Yoga"),
    location: asciiBytes("McCarren Park"),
    startsAtMs: 1784383200000,
    endsAtMs: 1784386800000,
    timing: "timed",
    eventKind: "personal",
    participation: "accepted",
    availability: "free",
    surfaces: { menuBar: "inherit", desktopWidget: "inherit", upNext: "inherit" },
    source: null,
  },
  {
    id: 109,
    calendarId: 3,
    title: asciiBytes("Weekly reset"),
    location: asciiBytes("Home"),
    startsAtMs: 1784494800000,
    endsAtMs: 1784498400000,
    timing: "timed",
    eventKind: "task",
    participation: "accepted",
    availability: "busy",
    surfaces: { menuBar: "inherit", desktopWidget: "inherit", upNext: "pin" },
    source: null,
  },
  {
    id: 110,
    calendarId: 4,
    title: asciiBytes("Pay card"),
    location: asciiBytes(""),
    startsAtMs: 1784073600000,
    endsAtMs: 1784160000000,
    timing: "all_day",
    eventKind: "task",
    participation: "accepted",
    availability: "free",
    surfaces: { menuBar: "hide", desktopWidget: "hide", upNext: "hide" },
    source: null,
  },
];

export const WIDGET_RULE: SurfaceRule = {
  surface: "desktop_widget",
  maxEvents: 3,
  horizonMinutes: 10080,
  includeAllDay: false,
  includeDeclined: false,
};

export const WEEK_DAYS_PREVIOUS: readonly WeekDayHeader[] = [
  { id: 1, name: asciiBytes("MON"), day: 6, today: false },
  { id: 2, name: asciiBytes("TUE"), day: 7, today: false },
  { id: 3, name: asciiBytes("WED"), day: 8, today: false },
  { id: 4, name: asciiBytes("THU"), day: 9, today: false },
  { id: 5, name: asciiBytes("FRI"), day: 10, today: false },
  { id: 6, name: asciiBytes("SAT"), day: 11, today: false },
  { id: 7, name: asciiBytes("SUN"), day: 12, today: false },
];

export const WEEK_DAYS_CURRENT: readonly WeekDayHeader[] = [
  { id: 1, name: asciiBytes("MON"), day: 13, today: false },
  { id: 2, name: asciiBytes("TUE"), day: 14, today: false },
  { id: 3, name: asciiBytes("WED"), day: 15, today: true },
  { id: 4, name: asciiBytes("THU"), day: 16, today: false },
  { id: 5, name: asciiBytes("FRI"), day: 17, today: false },
  { id: 6, name: asciiBytes("SAT"), day: 18, today: false },
  { id: 7, name: asciiBytes("SUN"), day: 19, today: false },
];

export const WEEK_DAYS_NEXT: readonly WeekDayHeader[] = [
  { id: 1, name: asciiBytes("MON"), day: 20, today: false },
  { id: 2, name: asciiBytes("TUE"), day: 21, today: false },
  { id: 3, name: asciiBytes("WED"), day: 22, today: false },
  { id: 4, name: asciiBytes("THU"), day: 23, today: false },
  { id: 5, name: asciiBytes("FRI"), day: 24, today: false },
  { id: 6, name: asciiBytes("SAT"), day: 25, today: false },
  { id: 7, name: asciiBytes("SUN"), day: 26, today: false },
];

export const HOUR_ROWS: readonly HourRow[] = [
  { id: 6, label: asciiBytes("6 AM") },
  { id: 7, label: asciiBytes("7 AM") },
  { id: 8, label: asciiBytes("8 AM") },
  { id: 9, label: asciiBytes("9 AM") },
  { id: 10, label: asciiBytes("10 AM") },
  { id: 11, label: asciiBytes("11 AM") },
  { id: 12, label: asciiBytes("12 PM") },
  { id: 13, label: asciiBytes("1 PM") },
  { id: 14, label: asciiBytes("2 PM") },
  { id: 15, label: asciiBytes("3 PM") },
  { id: 16, label: asciiBytes("4 PM") },
  { id: 17, label: asciiBytes("5 PM") },
  { id: 18, label: asciiBytes("6 PM") },
  { id: 19, label: asciiBytes("7 PM") },
  { id: 20, label: asciiBytes("8 PM") },
  { id: 21, label: asciiBytes("9 PM") },
];

export const WEEK_EVENT_BLOCKS: readonly WeekEventBlock[] = [
  { eventId: 101, calendarId: 1, dayIndex: 0, gapBefore: 216, height: 72, title: asciiBytes("Weekly planning"), timeLabel: asciiBytes("9:00–10:00"), location: asciiBytes("Desk"), tone: "surface", pinned: false },
  { eventId: 102, calendarId: 1, dayIndex: 1, gapBefore: 288, height: 72, title: asciiBytes("Design review"), timeLabel: asciiBytes("10:00–11:00"), location: asciiBytes("Studio"), tone: "surface", pinned: false },
  { eventId: 103, calendarId: 3, dayIndex: 2, gapBefore: 288, height: 132, title: asciiBytes("Deep work — calendar layout"), timeLabel: asciiBytes("10:00–12:00"), location: asciiBytes("Focus mode"), tone: "accent", pinned: true },
  { eventId: 104, calendarId: 2, dayIndex: 2, gapBefore: 144, height: 72, title: asciiBytes("Dentist"), timeLabel: asciiBytes("2:00–3:00"), location: asciiBytes("Greenpoint Dental"), tone: "muted", pinned: false },
  { eventId: 105, calendarId: 1, dayIndex: 3, gapBefore: 288, height: 72, title: asciiBytes("Project review"), timeLabel: asciiBytes("10:00–11:00 · tentative"), location: asciiBytes("Meet"), tone: "surface", pinned: false },
  { eventId: 106, calendarId: 3, dayIndex: 4, gapBefore: 216, height: 132, title: asciiBytes("Ship the first native pass"), timeLabel: asciiBytes("9:00–11:00"), location: asciiBytes("Focus mode"), tone: "accent", pinned: true },
  { eventId: 107, calendarId: 2, dayIndex: 4, gapBefore: 576, height: 96, title: asciiBytes("Dinner with Maya"), timeLabel: asciiBytes("7:00–9:00"), location: asciiBytes("Win Son"), tone: "muted", pinned: false },
  { eventId: 108, calendarId: 2, dayIndex: 5, gapBefore: 288, height: 72, title: asciiBytes("Yoga"), timeLabel: asciiBytes("10:00–11:00"), location: asciiBytes("McCarren Park"), tone: "muted", pinned: false },
  { eventId: 109, calendarId: 3, dayIndex: 6, gapBefore: 792, height: 72, title: asciiBytes("Weekly reset"), timeLabel: asciiBytes("5:00–6:00"), location: asciiBytes("Home"), tone: "surface", pinned: true },
];

export const MONTH_CELLS: readonly MonthCell[] = [
  { id: 1, day: 29, inMonth: false, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 2, day: 30, inMonth: false, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 3, day: 1, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 4, day: 2, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 5, day: 3, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Independence Day observed"), hasEvent2: false, event2: asciiBytes("") },
  { id: 6, day: 4, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 7, day: 5, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 8, day: 6, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 9, day: 7, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Call Mom"), hasEvent2: false, event2: asciiBytes("") },
  { id: 10, day: 8, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 11, day: 9, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Writing block"), hasEvent2: false, event2: asciiBytes("") },
  { id: 12, day: 10, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 13, day: 11, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 14, day: 12, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 15, day: 13, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Weekly planning"), hasEvent2: false, event2: asciiBytes("") },
  { id: 16, day: 14, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Design review"), hasEvent2: false, event2: asciiBytes("") },
  { id: 17, day: 15, inMonth: true, today: true, hasEvent1: true, event1: asciiBytes("Deep work"), hasEvent2: true, event2: asciiBytes("Dentist") },
  { id: 18, day: 16, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Project review"), hasEvent2: false, event2: asciiBytes("") },
  { id: 19, day: 17, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Ship first pass"), hasEvent2: true, event2: asciiBytes("Dinner with Maya") },
  { id: 20, day: 18, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Yoga"), hasEvent2: false, event2: asciiBytes("") },
  { id: 21, day: 19, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Weekly reset"), hasEvent2: false, event2: asciiBytes("") },
  { id: 22, day: 20, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 23, day: 21, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Research"), hasEvent2: false, event2: asciiBytes("") },
  { id: 24, day: 22, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 25, day: 23, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Therapy"), hasEvent2: false, event2: asciiBytes("") },
  { id: 26, day: 24, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 27, day: 25, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Farmers market"), hasEvent2: false, event2: asciiBytes("") },
  { id: 28, day: 26, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 29, day: 27, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Plan August"), hasEvent2: false, event2: asciiBytes("") },
  { id: 30, day: 28, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 31, day: 29, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Deep work"), hasEvent2: false, event2: asciiBytes("") },
  { id: 32, day: 30, inMonth: true, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 33, day: 31, inMonth: true, today: false, hasEvent1: true, event1: asciiBytes("Month review"), hasEvent2: false, event2: asciiBytes("") },
  { id: 34, day: 1, inMonth: false, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
  { id: 35, day: 2, inMonth: false, today: false, hasEvent1: false, event1: asciiBytes(""), hasEvent2: false, event2: asciiBytes("") },
];
